cmd/frontend/internal/auth/openidconnect/user_test.go

```diff
@@ -77,6 +77,7 @@ func TestAllowSignup(t *testing.T) {
 				},
 				&userClaims{},
 				test.usernamePrefix,
+				nil,
```

Comment:
When dealing with 10+ params, it's super-useful to call out what this stray `nil` is referring to.

```suggestion
				nil,  // UserCreateEventProperties
```